randovania/games/samus_returns/logic_database/Area 1.txt

```diff
@@ -939,19 +993,21 @@ Extra - asset_id: collision_camera_033
                       # https://youtu.be/41wPJZOtWfg
                       Damage Boost (Intermediate) and Normal Damage ≥ 10
               All of the following:
-                  Spider Ball
-                  Spring Ball or Mid-Air Morph (Beginner)
+                  # Spider Ball up
+                  Spider Ball and Mid-Air Morph (Beginner)
                   # Get enough height to use Spider Ball
                   High Jump Boots or Super Jump (Intermediate)
               Spider Boost (Intermediate) and Can Spider Boost
+              # Jump up with only HJ
+              High Jump Boots and Super Jump (Advanced)
 
 > Pickup (Super Missile Tank); Heals? False
   * Layers: default
   * Pickup 30; Category? Minor
   * Extra - actor_name: HiddenPowerup002
   * Extra - actor_type: item_supermissiletank
   > Door to Gullugg Gangway
-      Trivial
+      Gravity Suit and Varia Suit
```

Comment:
I'd remove Varia from this since the connection to this node never checks for Varia.